docs/source/intro/about.rst

```diff
@@ -4,17 +4,17 @@
 What is AiiDA
 *************
 
-Implement data-driven and fully automated workflows with AiiDA.
+AiiDA is an open-source Python infrastructure to automate, manage, persist, search and reproduce the complex workflows associated with modern computational science, and their associated data.
 
-AiiDA's primary features include:
+AiiDA is built to support and streamline its four core pillars of Automation, Data, Environment, and Sharing (the ADES model, described `here <https://arxiv.org/abs/1504.01163>`__). Some of the key features of AiiDA include:
 
-*  **Workflows:** Write complex, auto-documenting workflows in Python, linked to arbitrary executables on local and remote computers.
-   The event-based workflow engine supports tens of thousands of processes per hour with full check-pointing.
-*  **Data provenance:** Automatically track inputs, outputs & metadata of all calculations in a provenance graph for full reproducibility.
-   Perform fast queries on graphs containing millions of nodes.
-*  **HPC interface:** Move your calculations to a different computer by changing one line of code. AiiDA is compatible with schedulers like `SLURM <https://slurm.schedmd.com>`__, `PBS Pro <https://www.pbspro.org/>`__, `torque <http://www.adaptivecomputing.com/products/torque/>`__, `SGE <http://gridscheduler.sourceforge.net/>`__ or `LSF <https://www.ibm.com/support/knowledgecenter/SSETD4/product_welcome_platform_lsf.html>`__ out of the box.
-*  **Plugin interface:** Extend AiiDA with `plugins <https://aiidateam.github.io/aiida-registry/>`__ for new simulation codes (input generation & parsing), data types, schedulers, transport modes and more.
-*  **Open Science:** Export subsets of your provenance graph and share them with peers or make them available online for everyone on the `Materials Cloud <https://www.materialscloud.org>`__.
-*  **Open source:** AiiDA is released under the `MIT open source license <LICENSE.txt>`__
+*  **Workflows:** AiiDA allows to build and execute complex, auto-documenting workflows linked to multiple codes on local and remote computers.
+*  **High-throughput:** AiiDA’s event-based workflow engine supports tens of thousands of processes per hour with full check-pointing.
+*  **Data provenance:** AiiDA automatically tracks and records inputs, outputs and metadata of all calculations in extensive provenance graphs that preserve and persist the full lineage of all parent and child data and calculations.
```

Comment:
what is the difference between "preserve" and "persist"? Feels a bit like a tautology to me. 
Also, we should probably change "calculations" for "workflows and calculations" or just "workflows" since that is described in the first point.

Also: "lineage of all parent and child data and calculations." I don't find very clear. I think "full lineage of all data" captures exactly the same and is a lot clearer.

Reply:
Ok, I tried to adapt the text to address the various comments.